### DISCUSSION TITLE
No matter what I do, I can't get verified

### DISCUSSION THREAD

**Discussion by smborder** [community/community#118595] [GitHub Education]
### Hi everyone,

I used my school email. 

I tried to upload a smartphone pic of my proof document since my webcam is pretty bad, and got this error:

> Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

So then I used my webcam a total of 4 times and got both of the following errors:

> Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.

> Please ensure that your academic affiliation document contains your first name exactly as it appears in your GitHub billing information. Variations of your name and nicknames are unacceptable.

My document is a Statement of Academic Record with info on my current coursework as of April 4th 2024, and it addresses me as Miss Firstname Lastname. I don't really know what other document I could provide...

What do I do now?

**Comment by wiseken** [comment]
hmmm

**Comment by samus-aran** [comment]
Hi @smborder 

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

Hope this helps 👍
